makedoc.g

```diff
@@ -7,18 +7,22 @@ if fail = LoadPackage("AutoDoc", "2019.04.10") then
     Error("AutoDoc version 2019.04.10 or newer is required.");
 fi;
 
-AutoDoc( 
-        rec(
-            scaffold := rec( gapdoc_latex_options := rec( 
-                             LateExtraPreamble := "\\usepackage{amsmath}\\usepackage[T1]{fontenc}\n\\usepackage{tikz}\n\\usetikzlibrary{shapes,arrows,matrix}\n\\usepackage{faktor}" 
-                                                        ),
-                             entities := [ "GAP4", "CAP" ],
-                             ),
-            
+AutoDoc(rec(
+            scaffold := rec(
+                gapdoc_latex_options := rec(
+                LateExtraPreamble := """
```

Comment:
Should this be indented one level more? (I'm not sure what the intedentation standard for `rec`s with only a single entry is.)

Reply:
There aren't really any such standards, and this file never followed any, but I agree this should be formatted differently. I'll update it.